src/main/java/org/betonquest/betonquest/events/ExperienceEvent.java

```diff
@@ -3,17 +3,21 @@
 import org.betonquest.betonquest.Instruction;
 import org.betonquest.betonquest.VariableNumber;
 import org.betonquest.betonquest.api.QuestEvent;
+import org.betonquest.betonquest.api.profiles.OnlineProfile;
 import org.betonquest.betonquest.api.profiles.Profile;
 import org.betonquest.betonquest.exceptions.InstructionParseException;
 import org.betonquest.betonquest.exceptions.QuestRuntimeException;
-import org.bukkit.entity.Player;
 
 /**
  * Gives the player specified amount of experience
  */
 @SuppressWarnings("PMD.CommentRequired")
 public class ExperienceEvent extends QuestEvent {
 
+    /**
+     * The experience level the player needs to get.
+     * The decimal part of the number is a percentage of the next level.
+     */
     private final VariableNumber amount;
     private final boolean checkForLevel;
```

Comment:
This boolean's name does not describe it's functionality.

Reply:
it does not? in my opinion it does.